internal/functions/efgs/configuration.go

```diff
@@ -31,6 +31,8 @@ type publishConfig struct {
 	KeyServer          *utils.KeyServerConfig
 	Client             *http.Client
 	MaxBatchSize       int `env:"MAX_UPLOAD_KEYS,default=30"`
+	MaxKeyAge          int `env:"MAX_KEY_AGE,default=15"`
+	MaxOverlappingKeys int `env:"MAX_SAME_DAY_KEYS,default=15"`
```

Comment:
Can we keep the naming 1:1 with upstream? :D
https://github.com/google/exposure-notifications-server/blob/b3ac17696bd9c344fa9b4ba7eaac150a3d886d68/internal/generate/config.go#L44
https://github.com/google/exposure-notifications-server/blob/b3ac17696bd9c344fa9b4ba7eaac150a3d886d68/internal/generate/config.go#L45